### ISSUE TITLE
weight decay in add_weights_regularizer

### ISSUE THREAD

**Issue by igorbpf** [tflearn/tflearn#678]
Is weight_decay parameter in add_weights_regularizer the regularization constrain?

**Comment by arashsaber** [comment]
For example L1 loss is 
    L1loss = tf.reduce_sum(tf.abs(W)), 
then for weight_decay wd, the total los is
    tf.multiply(L1loss,wd)
In short it is just a weight (scaling) for the loss
